packages/react/src/profiler.tsx

```diff
@@ -36,7 +38,7 @@ function pushActivity(name: string, op: string): number | null {
 
   return (globalTracingIntegration as any).constructor.pushActivity(name, {
     description: `<${name}>`,
-    op: `react.${op}`,
+    op: `${REACT_OP}.${op}`,
```

Comment:
Should we modify `pushActivity` to take the entire op string? Then pass it the const?

Reply:
Yes we should!